soc/sifive/sifive_freedom/fu500/Kconfig

```diff
@@ -18,3 +18,9 @@ config SOC_SERIES_SIFIVE_FREEDOM_FU500
 
```

Comment:
not part of your commit but drop this empty line

Reply:
Should be fixed now